api/src/main/java/ca/bc/gov/educ/api/grad/report/service/CommonService.java

```diff
@@ -535,8 +538,8 @@ private void processCertificate(List<List<UUID>> partitions, StudentSearchReques
                         gradStudentCertificatesRepository.findRecordsForUserRequestByStudentIdOnly(subList);
             } else {
                 scdSubList = onlyWithNullDistributionDate?
-                        gradStudentTranscriptsRepository.findRecordsWithNullDistributionDateForUserRequest(subList)
-                        : gradStudentTranscriptsRepository.findRecordsForUserRequest(subList);
+                        gradStudentCertificatesRepository.findRecordsWithNullDistributionDateForUserRequest(subList)
```

Comment:
just double check this is your intention to switch gradStudentTranscriptsRepository to gradStudentCertificateRepository in processTranscript method?

Reply:
I fixed the original copy/paste issue. Please open the history of the class to see that it was gradStudentTranscriptsRepository in the processCertificate method. I fixed that.